pages/work-with-us/index.tsx

```diff
@@ -1,26 +1,80 @@
-import { useState, useEffect } from 'react'
+import { Heading, Stack, Text, Box, VStack } from '@chakra-ui/react'
 import IntroductionComponent from 'components/work-with-us/introduction'
-import carouselsData from '../../assets/content/pages/work-with-us/carouselContent.json'
-import Carousel from 'components/work-with-us/Carousel'
-import { ulid } from 'ulid'
+import Section from 'components/_common/section'
+import { WORK_WITH_US } from 'constants/data'
 
-export default function WorkWithUsPage() {
-  // TODO : refactor this into a hook
-  const [width, setWidth] = useState(0)
-  useEffect(() => {
-    setWidth(document.body.clientWidth)
-  }, [])
+const SectionTitle = ({
+  title,
+  description,
+}: {
+  title: string
+  description: string
+}): React.ReactElement => (
+  <Stack
+    as={Box}
+    textAlign={'center'}
+    spacing={{ base: 3, md: 5 }}
+    py={{ sm: 10, md: 18 }}
+    my={{ base: 10, sm: 10, md: 20 }}>
+    <Heading
+      fontWeight={600}
+      fontSize={{ base: '2xl', sm: '3xl', md: '4xl' }}
+      lineHeight={'110%'}
+      color={'white.100'}
+      fontFamily='Clear Sans'>
+      {title}
+    </Heading>
+    <Text
+      color='grey'
+      fontSize={{ base: 'xl', lg: '2xl' }}
+      fontFamily='Clear Sans'>
+      {description}
+    </Text>
+  </Stack>
+)
 
-  typeof window !== 'undefined' &&
-    window.addEventListener('resize', function () {
-      setWidth(document.body.clientWidth)
-    })
+const WORK_WITH_US_CATEGORIES = [
+  {
+    title: 'Consulting and Education',
+    description: 'Learn from Web3 experts',
+    categorySlug: 'consulting-and-education',
+  },
+  {
+    title: 'Sponsorships and Marketing',
+    description: 'Scale your presence in Web3',
+    categorySlug: 'sponsorships-and-marketing',
+  },
+  {
+    title: 'Creative Services',
+    description: 'Tell your story and reach a new audience',
+    categorySlug: 'creative-services',
+  },
+  {
+    title: 'Software and DAO tooling',
+    description: 'Level up your organization with cutting-edge tooling',
+    categorySlug: 'software-and-dao-tooling',
+  },
+]
 
+export default function WorkWithUsPage() {
   return (
     <>
       <IntroductionComponent />
-      {carouselsData.map((carousel, index) => (
-        <Carousel width={width} carouselIndex={index} {...carousel} key={ulid()} />
+      {WORK_WITH_US_CATEGORIES.map((category, index) => (
+        <Box key={`category-${index}`}>
+          <SectionTitle title={category.title} description={category.title} />
```

Comment:
Should be `category.description`
![image](https://github.com/BanklessDAO/bankless-website/assets/2530913/fecc3259-4754-4dc4-8531-12673f84ad5a)


```suggestion
<SectionTitle title={category.title} description={category.description} />
```